docs/articles/baseplatform/reproducible-yocto-builds.rst

```diff
@@ -0,0 +1,90 @@
+Reproducible Yocto builds
+=========================
+
+Intro
+-----
+
+A bitbake recipe specifies where the source code of a package should be downloaded from. Because there are several ways to do this, a recipe might not specify a specific version but rather the head of some branch or a latest release.
```

Comment:
This does not make sense to have first in the intro. Shouldn't the intro be about why and a broad overview of how, rather than why there can be issues with this?

Of course, the issue with AUTOREV should be mentioned somewhere, but perhaps not in the introduction.